tests/unit/test_generators.py

```diff
@@ -0,0 +1,18 @@
+import numpy as np
+
+from jina.types.document.generators import from_ndarray
+
+def test_from_ndarray():
+    test_array = np.random.random((10, 64))
+    # yield using from_ndarray along axis 0
+    test_generator = from_ndarray(test_array)
+    assert next(iter(test_generator)).blob.shape == np.array([64])
+
+    # yield using from_ndarray along axis 1
+    test_array = test_array.T
+    test_generator = from_ndarray(test_array, axis=1)
```

Comment:
shouldnt here the shape be 10?

Reply:
I see now, the transpose. May u parametrize better the input and the expected shape? this way the test would be more readable?

Reply:
yes, the transpose can be missed. Parametrized like this it's maybe more explicit.